client/blocks/image-selector/index.jsx

```diff
@@ -59,8 +58,11 @@ export class ImageSelector extends Component {
 		const { siteId, imageIds } = this.props;
 
 		if ( imageIds ) {
-			const images = imageIds.map( ( imageId ) => MediaStore.get( siteId, imageId ) );
-			this.props.setMediaLibrarySelectedItems( siteId, images );
+			this.props.setMediaLibrarySelectedItems(
+				siteId,
+				// this action only cares about the ID so there's no reason to retrieve the full image object from the store
```

Comment:
(not blocking) nit: this comment makes sense to me from a reviewers' perspective. However, if one comes back to the code later it's maybe not obvious without looking at the git history of the file. Would it make sense to you to keep this comment as part of the commit message instead?